### DISCUSSION TITLE
Bitbucket Pipelines

### DISCUSSION THREAD

**Discussion by marcossv9** [github/gh-actions-importer#32] [Feedback] [answered]
Hey folks, is there any timeline or roadmap, where we can check future integrations?

I'm specifically interested in the timeline to support the migration from Bitbucket Pipelines to Github Actions.

Thanks,

**Comment by ethanis** [comment, marked as answer]
@marcossv9 thanks for your patience! At this time, we don't have plans to add a Bitbucket Pipelines migration path. We'll be publishing updates here, and on the public GitHub [roadmap](https://github.com/orgs/github/projects/4247), to help you stay up-to-date with what's coming to the product.
